### ISSUE TITLE
Errors in STDERR are not displayed

### ISSUE THREAD

**Issue by jyggen** [phpro/grumphp#141]
Certain errors for certain tasks are not printed to `STDOUT` but rather `STDERR` which means they won't be displayed in GrumPHP's error output (since all tasks exclusively use `Symfony\Component\Process\Process::getOutput()`).

An example is the error mentioned in #140; with output only taken from `STDOUT` it won't really tell me what went wrong:

```console
GrumPHP detected a pre-commit command.
GrumPHP is sniffing your code!
Running task  1/10: Composer
Running task  2/10: JsonLint
Running task  3/10: XmlLint
Running task  4/10: YamlLint
Running task  5/10: PhpLinterTask
Running task  6/10: Phpunit
Aborted ...
 _____     _ _                _
|  ___|_ _(_) |_   _ _ __ ___| |
| |_ / _` | | | | | | '__/ _ \ |
|  _| (_| | | | |_| | | |  __/_|
|_|  \__,_|_|_|\__,_|_|  \___(_)
```

Changing so it uses `Symfony\Component\Process\Process::getErrorOutput()` instead will actually tell us what's up:

```console
GrumPHP detected a pre-commit command.
GrumPHP is sniffing your code!
Running task  1/10: Composer
Running task  2/10: JsonLint
Running task  3/10: XmlLint
Running task  4/10: YamlLint
Running task  5/10: PhpLinterTask
Running task  6/10: Phpunit
Aborted ...
 _____     _ _                _
|  ___|_ _(_) |_   _ _ __ ___| |
| |_ / _` | | | | | | '__/ _ \ |
|  _| (_| | | | |_| | | |  __/_|
|_|  \__,_|_|_|\__,_|_|  \___(_)

PHP Fatal error:  Call to undefined method PHPUnit_TextUI_TestRunner::printVersionString() in /usr/share/php/PHPUnit/TextUI/Command.php on line 816
```

So it would probably be a good idea to fetch output from both streams. I could probably PR something if you want.

Thanks!

**Comment by veewee** [comment]
This sure is an issue. I also encountered this one in the past.
Currently all the tasks just use the STDOUT output which works fine in most cases.
Maybe it's a better idea to fall back to the error message if no stdout message is set? 
This logic could be placed in a separate location so that it is the same for every task. 
A while ago, I was thinking about creating some Formatters (#11), like in this case a `RawProcessOutputFormatter`. The interface could be different or maybe you find a better solution.

Feel free to create a proposal or a PR!

**Comment by countless-integers** [comment]
This would explain why I'm getting only ascii art in my output and empty lines where the actual errors are supposed to be...

**Comment by veewee** [comment]
Fixed in #142
